src/main/java/org/akaza/openclinica/view/Page.java

```diff
@@ -635,6 +635,9 @@
     JOBS ("/WEB-INF/jsp/" + "managestudy/jobs.jsp", "jobs (bulk logfile)"),
     JOBS_SERVLET ("/Jobs", "jobs (bulk logfile)"),
 
+    INSIGHT ("/WEB-INF/jsp/" + "managestudy/embed-insight.jsp", "embedded insight reports"),
```

Comment:
rename the jsp to something like "embedded-reports.jsp"

Reply:
ok renamed